src/main/java/org/opentripplanner/transit/raptor/rangeraptor/standard/stoparrivals/Stops.java

```diff
@@ -36,12 +36,19 @@ public void setupEgressStopStates(
     ) {
         for (int round = 1; round < stops.length; round++) {
             for (RaptorTransfer egressPath : egressPaths) {
-                EgressStopArrivalState<T> state = new EgressStopArrivalState<>(
+                if(stops[round][egressPath.stop()] == null) {
+                    EgressStopArrivalState<T> state = new EgressStopArrivalState<>(
                         round,
                         egressPath,
                         transitArrivalCallback
-                );
-                stops[round][egressPath.stop()] = state;
+                    );
+                    stops[round][egressPath.stop()] = state;
+                }
+                else {
+                    throw new IllegalStateException(
+                        "State exist for stop: " + egressPath.stop() + ", round: " + round
```

Comment:
Expand or change exception message to explain in terms of an unimplemented feature.

Reply:
I added more doc to this.